src/tests/Hspec.hs

```diff
@@ -10,7 +11,9 @@ import           IHaskell.Test.Eval (testEval)
 import           IHaskell.Test.Hoogle (testHoogle)
 
 main :: IO ()
-main =
+main = do
+  setLocaleEncoding utf8
```

Comment:
Might be good to put a comment here explaining why we're doing this for when we inevitably forget a few months from now.

Reply:
Will do if it works!

Reply:
Hmm, this isn't producing identical behavior with my Docker container yet.

## CI

```
  src/tests/IHaskell/Test/Eval.hs:49: 
  1) Code Evaluation evaluates :in directive
       Expected display datas to be "[[{\"text/plain\":\"type String = [Char] \\t-- Defined in \195\162\194\128\194\152GHC.Base\195\162\194\128\194\153\"}]]". Got "[[{\"text/plain\":\"type String = [Char] \\t-- Defined in \226\128\152GHC.Base\226\128\153\"}]]"
```

## Docker

```
ihaskell              >   src/tests/IHaskell/Test/Eval.hs:49: 
ihaskell              >   2) Code Evaluation evaluates :in directive
ihaskell              >        Expected display datas to be "[[{\"text/plain\":\"type String = [Char] \\t-- Defined in \195\162\194\128\194\152GHC.Base\195\162\194\128\194\153\"}]]". Got "[[{\"text/plain\":\"type String = [Char] \\t-- Defined in `GHC.Base'\"}]]"
```

It hurts my brain a little to think about where this discrepancy could be arising...

Reply:
Current hypotheses:

* GHC is called as a library within IHaskell somehow, maybe this doesn't respect the current process's locale and needs to be configured some other way?
* Something about how the logs are processed in GitHub Actions is changing it on the CI side?? 

Maybe next step is to try `LC_ALL=C` vs `LC_ALL=C.UTF-8`

Reply:
I reverted that commit since it seems to have no effect.